content/en/_index.md

```diff
@@ -337,6 +388,10 @@ function example_snippet(string $projectId, string $filePath): void
     );
 }
 {{< /tab >}}
+{{< tab header="Terraform" >}}
+Terraform doesn't have custom, user-defined methods or functions.
```

Comment:
Make this a comment. Markdown url's don't render in tabs, so just include the URL

Reply:
done